style.css

```diff
@@ -61,3 +62,148 @@ a {
   color: inherit;
   text-decoration: none;
 }
+
+.sec2-title {
+  margin-top: 80px;
+  margin-bottom: 60px;
+  border-bottom: 1px solid grey;
+  padding-bottom: 30px;
+  text-align: center;
+  font-family: "Crete Round", serif;
+  width: 300px;
+  font-size: 40px;
+}
+
+.sec2 {
+  display: flex;
+  justify-content: center;
+}
+
+.sec3 {
+  display: grid;
+  place-items: center;
+  padding: 15px;
+}
+
+.card1 > h2 {
+  font-family: "Crete Round", serif;
+  margin-top: 20px;
+  margin-bottom: 30px;
+  font-size: 30px;
+}
+
+.box {
+  display: flex;
+  margin: 10px 0;
+}
+
+.box > p {
+  padding: 5px;
+  border: 1px solid black;
+  margin-right: 10px;
+}
+
+.button1 {
+  background-color: rgba(255, 107, 0, 1);
+  color: white;
+  border: none;
+  padding: 10px;
+}
+
+.card2 {
+  position: relative;
+  margin-top: 30px;
+}
+
+.card-text {
+  position: absolute;
+  bottom: 40px;
+  padding: 30px;
+  color: white;
+}
+
+.box2 {
+  display: flex;
+  margin-top: 15px;
+}
+
+.box2 > p {
+  background-color: rgba(255, 255, 255, 0.24);
+  padding: 5px;
+  margin-right: 10px;
+  font-family: 'Inter', sans-serif;
+}
+
+.card-text > h2 {
+  font-family: "Crete Round", serif;
+  font-size: 30px;
+  margin-bottom: 15px;
+}
+
+.card-text > p {
+  font-family: 'Inter', sans-serif;
+}
+
+.button2 {
+  background-color: rgba(255, 107, 0, 1);
+  color: white;
+  border: none;
+  padding: 10px;
+  width: 330px;
+}
+
+.about {
+  background-color: #1c1a19;
+  background-image: url("resources/Frame.png");
+  background-repeat: no-repeat;
+  background-position: bottom right;
+  height: 90vh;
+  padding: 30px;
+  margin-top: 90px;
```

Comment:
- [ ] Kindly check through your project to ensure you used the right spacing for each element.